jellyfinclient/src/actions/LoginAction.js

```diff
@@ -2,21 +2,23 @@ import * as types from "./ActionTypes";
 
 export default function LoginAction(serverAddress, username, password) {
     return (dispatch) => {
-        console.log("Logging in with:", serverAddress, username, password)
-        dispatch(loginSuccessfully({serverAddress: serverAddress, username: username, password: password}))       
+        console.log("Logging in with:", serverAddress, username, password);
+        return (dispatch(loginSuccessfully(username, password)));
     }
 }
 
-function loginSuccessfully(payload) {
+function loginSuccessfully(username, password) {
```

Comment:
Think of future cases, where you would like to add more values to pass on login (maybe): Then payload would make absolutely sense as it can be easily adapted, whereas for arguments you have to modify all method signatures, ...

(just my two cents) 😄

Reply:
Generally you are right, but I wouldn't store variables in the persist storage before checking them. This will prevent accidentally storing critical data in redux.